src/components/ProjectsPage.js

```diff
@@ -1,83 +1,117 @@
 import React from "react";
+import { Button, Row, Col, Card, Container } from 'react-bootstrap';
+import "../styling/projects-page-styling.css";
+
+const { Body, Title, Subtitle, Text, Link } = Card;
+const { Fragment } = React;
+
+let projects = [{id: 0, title: "Project Title 1", company: "Organization Name 1"},
+                 {id: 1, title: "Project Title 2", company: "Organization Name 2"},
+                 {id: 2, title: "Project Title 3", company: "Organization Name 3"},
+                 {id: 3, title: "Project Title 4", company: "Organization Name 4"},
+                 {id: 4, title: "Project Title 5", company: "Organization Name 5"},
+                 {id: 5, title: "Project Title 6", company: "Organization Name 6"},
+                 {id: 6, title: "Project Title 7", company: "Organization Name 7"}]
 
-let companies = [{id: 0, title: "Project Title 1", name: "Organization Name 1"},
-                 {id: 1, title: "Project Title 2", name: "Organization Name 2"},
-                 {id: 2, title: "Project Title 3", name: "Organization Name 3"},
-                 {id: 3, title: "Project Title 4", name: "Organization Name 4"},
-                 {id: 4, title: "Project Title 5", name: "Organization Name 5"}]
 
 class ProjectsPage extends React.Component {
+  state = {
+    selected: -1
+  }
+
+  render() {
+    return (
+      <Fragment>
+        { this.state.selected === -1 &&
+        <Container className="pg-container">
+          <Row className="d-flex justify-content-center">
+            <Row className="d-flex justify-content-left">
+              {projects.map(project =>
+                <Col lg={4}>
+                  <Card className="broad card-btn" onClick={() => this.setState({selected: project.id})}>
```

Comment:
could we add an animation so the transition between the views is less abrupt? i think you can do this by messing around with the CSS properties transition, animation, etc.